PProPanel/index.html

```diff
@@ -22,15 +21,26 @@
            	csInterface.evalScript('$._ext_PPRO.getActiveSequenceName()', myCallBackFunction);  
         });
 
+        $("#checkforums").on("click", function(e){
+            e.preventDefault(); 
+            var csInterface = new CSInterface();
+            csInterface.openURLInDefaultBrowser("https://forums.adobe.com/community/premiere/sdk");
+        });
+
         $("#newseqfrompreset").on("click", function(e){
            	e.preventDefault(); 
            	var csInterface = new CSInterface();
-           	var path = csInterface.getSystemPath(SystemPath.EXTENSION);
-
-           	if (path != null)
-           	{
-           		path = path + '/payloads/PProPanel.sqpreset';
-           	}
+			var OSVersion   = csInterface.getOSInformation();
+            var path    	= csInterface.getSystemPath(SystemPath.EXTENSION);
+
+			if (OSVersion != null){
+				if (OSVersion.indexOf("Windows") >=0){
+					var sep = '\\\\';
+					path = path.replace(/\//g, sep);
+            } else {
+              var sep = '/';
+            }
+           		path = path + sep + 'payloads' + sep + 'PProPanel.sqpreset';
```

Comment:
Fixed! I'd read something scary about += operators in JavaScript ages back, but that seems to work fine.